libnmstate/validator.py

```diff
@@ -109,6 +113,15 @@ def validate_link_aggregation_state(desired_state, current_state):
                 specified_slaves |= slaves
 
 
+def validate_unique_interface_name(state):
+    ifaces_names = [
+        ifstate[schema.Interface.NAME]
+        for ifstate in state.get(schema.Interface.KEY, [])
+    ]
+    if len(ifaces_names) != len(set(ifaces_names)):
+        raise NmstateDuplicateInterfaceNameError(str(sorted(ifaces_names)))
```

Comment:
The exception should use a user-friendly message instead of just listing interfaces.

Reply:
Done